narayana/ArjunaJTA/jta/tests/classes/com/arjuna/ats/jta/xa/performance/HQStoreBenchmark.java

```diff
@@ -16,20 +16,24 @@
  */
 package com.arjuna.ats.jta.xa.performance;
 
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.infra.Blackhole;
+
 import com.arjuna.ats.arjuna.common.CoreEnvironmentBeanException;
 import com.arjuna.ats.internal.arjuna.objectstore.hornetq.HornetqJournalEnvironmentBean;
 import com.arjuna.ats.internal.arjuna.objectstore.hornetq.HornetqObjectStoreAdaptor;
 import com.arjuna.common.internal.util.propertyservice.BeanPopulator;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.openjdk.jmh.annotations.*;
-
-import javax.transaction.TransactionManager;
 
 @State(Scope.Benchmark)
 public class HQStoreBenchmark extends JTAStoreBase {
 
-    private static TransactionManager tm;
+//    private static TransactionManager tm;
```

Comment:
We normally would not allow commenting out code without a comment why. In this particular instance, since your changes are for performance reasons, I would just delete this line altogether.

Reply:
I thought this line would be used in future by someone, since this member variable was not used, so I just thought to comment, and keep it.

I will remove this now.